tests/mocked/test_image_actions.py

```diff
@@ -0,0 +1,133 @@
+# pylint: disable=line-too-long
+# pylint: disable=duplicate-code
+"""Mock tests for the Image Actions API resource"""
+
+import responses
+
+from pydo import Client
+
+
+@responses.activate
+def test_image_actions_get(mock_client: Client, mock_client_url):
+    """Tests Retrieving an Existing Image Action"""
+    image_id = 7938269
+    action_id = 36805527
+    expected = {
+        "action": {
+            "id": action_id,
+            "status": "in-progress",
+            "type": "transfer",
+            "started_at": "2014-11-14T16:42:45Z",
+            "completed_at": None,
+            "resource_id": image_id,
+            "resource_type": "image",
+            "region": {
+                "name": "New York 3",
+                "slug": "nyc3",
+                "sizes": ["s-1vcpu-3gb", "m-1vcpu-8gb"],
+                "features": ["private_networking", "image_transfer"],
+                "available": True,
+            },
+            "region_slug": "nyc3",
+        }
+    }
+
+    responses.add(
+        responses.GET,
+        f"{mock_client_url}/v2/images/{image_id}/actions/{action_id}",
+        json=expected,
+    )
+
+    get_resp = mock_client.image_actions.get(image_id=image_id, action_id=action_id)
+    # TODO (to remove):
+    # 1. action_id is missing in openapi spec for Python in Request samples:
+    # https://docs.digitalocean.com/reference/api/api-reference/#operation/imageActions_get
+    # 2. image_id in Python Request samples is better be updated
+    # with the value in response - 7938269 - to avoid confusion
+    # Should we update openapi spec? Should the ticket be created for this update?
```

Comment:
Great catches! We should get those fixed up.

Reply:
Done in https://github.com/digitalocean/openapi/pull/910